src/app/components/keyboard/keyboard.component.ts

```diff
@@ -12,10 +14,19 @@ export class KeyboardComponent implements OnInit {
   @Output() onMouseDown = new EventEmitter<string>();
   @Output() onMouseUp = new EventEmitter<string>();
   ArrowButton = ArrowButton;//eslint-disable-line @typescript-eslint/naming-convention
+  pauseButtonLabel: string;
```

Comment:
Instead of `pauseButtonLabel: string`, I prefer to have  `pauseButtonLabel$: Observable<string>`

Reply:
Updated code as per comments